data/com.endlessm.SystemExternalApps.rules.in

```diff
@@ -0,0 +1,17 @@
+polkit.addRule(function(action, subject) {
+    if (action.id == "com.endlessm.SystemExternalApps.build-install" &&
+        action.lookup("user") == "@helper_user@" && subject.active == true &&
+	subject.local == true) {
+	    if (subject.isInGroup("@privileged_group@"))
+	        return polkit.Result.YES;
+	    else
+	        return polkit.Result.AUTH_ADMIN;
+    }
+    if ((action.id == "org.freedesktop.Flatpak.app-install" ||
+         action.id == "org.freedesktop.Flatpak.runtime-install" ||
+	 action.id == "org.freedesktop.Flatpak.runtime-uninstall") &&
```

Comment:
Looking at the actions, should the helper user be allowed to call `app-install` or `runtime-uninstall`? It seems like only `runtime-install` should be allowed since that's all the helper script is doing.